conflict (#290)
ComputerScienceHouse/Constitution: constitution.tex

Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
Any modifications made due to the discussion are added to the written proposal and the modified proposal is posted during the week.
The final proposal is presented the following week, and ballots are distributed for a Balloted Two-Thirds Vote with two-thirds quorum as described in \ref{Balloted Vote}.

> If two or more modifications are conflicting, ballots are distributed for a Balloted Vote with two-thirds quorum, and a winner is selected as described in \ref{Ranked Choice}.

Author: ```suggestion
If two or more modifications are conflicting such that they create a paradox within the constitution, ballots are distributed for a Balloted Vote with two-thirds quorum, and a winner is selected as described in \ref{Ranked Choice}.
```

Author: ```suggestion
If two or more modifications are conflicting such that they contradict one another, ballots are distributed for a Balloted Vote with two-thirds quorum, and a winner is selected as described in \ref{Ranked Choice}.
```

Author: ```suggestion
If two or more modifications would conflict with one another, at the discretion of the Maintainers, ballots are distributed for a Balloted Vote with two-thirds quorum, and a winner is selected as described in \ref{Ranked Choice}.
```

Reviewer: The third change here seems most appropriate.
